pkgs/development/python-modules/torch/default.nix

```diff
@@ -201,6 +201,7 @@ let
       MPISupport && cudaSupport && (mpi.cudatoolkit != cudaPackages.cudatoolkit);
     "Magma cudaPackages does not match cudaPackages" =
       cudaSupport && (effectiveMagma.cudaPackages != cudaPackages);
+    "Rocm support is currently broken because `rocmPackages.hipblaslt` is broken. (2024-06-09)" = rocmSupport;
```

Comment:
```
nix-repl> rocmPackages.hipblaslt
error: attribute 'hipblaslt' missing
       at «string»:1:1:
            1| rocmPackages.hipblaslt
             | ^
       Did you mean hipblas?
```

Super misleading, given that we don't even have it packaged at this time.

Reply:
I agree. I chose this wording because the linked comment suggested that the package was broken.
Maybe `unavailable` is a better term ?

Reply:
unpackaged is probably the correct term.